ui/api.py

```diff
@@ -53,3 +54,42 @@ def process_dropbox_data(dropbox_upload_data):
             "task": task_result.id,
         }
     return response_data
+
+
+def post_hls_to_edx(video_file):
+    """
+    Posts an HLS video to edX via API using attributes from a video file
+
+    Args:
+        video_file (ui.models.VideoFile): An HLS-encoded video file
+
+    Returns:
+        requests.models.Response: The API response
+    """
+    assert edx_settings_configured(), "edX settings need to be configured"
+    assert video_file.can_add_to_edx(), "This video file is not of the correct type"
```

Comment:
remove asserts?

Reply:
I just put this here as a bit of extra assurance. This function can be called from anywhere, and I wanted to make it explicit that it should only be used under these conditions

Reply:
👍